frontend/src/components/GoBack/GoBack.js

```diff
@@ -0,0 +1,32 @@
+import "./GoBack.scss";
+import React from "react";
+import { IconButton } from "@material-ui/core";
+import ConfirmationDialog from "../ConfirmationDialog/ConfirmationDialog";
+
+const defaultBack = (
+  <IconButton onClick={() => window.history.back()}>
+    <i className="fas fa-arrow-left" />
+  </IconButton>
+);
+
+const handleSuccess = () => {
+  window.history.back();
```

Comment:
This will cause the browser to refresh, possibly incorrectly. Use react-router history instead with `goBack` method. https://reacttraining.com/react-router/web/api/history

Reply:
Also no need for this higher order function, just pass the method directly as prop